[CI] IndexFollowingIT.testFollowIndexMaxOperationSizeInBytes failure on 7.6

Build link: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+matrix-java-periodic/ES_RUNTIME_JAVA=adoptopenjdk11,nodes=general-purpose/84/console

Build scan: https://gradle-enterprise.elastic.co/s/cncnkoxasahgs

Failure:
```
java.lang.AssertionError: 
incorrect global checkpoint {"remote_cluster":"leader_cluster","follow_shard_index":"index2","follow_shard_index_uuid":"PzWs2a3MRH6rPCRmfhkCOQ","follow_shard_shard":0,"leader_shard_index":"index1","leader_shard_index_uuid":"Hj6lFRfERAmQQtHOSR-u5A","leader_shard_shard":0,"max_read_request_operation_count":5120,"max_write_request_operation_count":5120,"max_outstanding_read_requests":12,"max_outstanding_write_requests":9,"max_read_request_size":"1b","max_write_request_size":"9223372036854775807b","max_write_buffer_count":2147483647,"max_write_buffer_size":"512mb","max_retry_delay":"10ms","read_poll_timeout":"10ms","headers":{}}
Expected: <998L>
     but: was <625L>

at __randomizedtesting.SeedInfo.seed([F195E07956D7667D:FF5106F7DA71098A]:0)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.junit.Assert.assertThat(Assert.java:956)
at org.elasticsearch.xpack.ccr.IndexFollowingIT.lambda$assertTask$63(IndexFollowingIT.java:1511)
at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:905)
at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:878)
at org.elasticsearch.xpack.ccr.IndexFollowingIT.testFollowIndexMaxOperationSizeInBytes(IndexFollowingIT.java:633)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:566)
```

Reproduce with:
```
./gradlew ':x-pack:plugin:ccr:internalClusterTest' --tests "org.elasticsearch.xpack.ccr.IndexFollowingIT.testFollowIndexMaxOperationSizeInBytes" -Dtests.seed=F195E07956D7667D -Dtests.security.manager=true -Dtests.locale=sq -Dtests.timezone=Asia/Aqtobe -Dcompiler.java=13
```

Could not reproduce locally.

--- a/x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
+++ b/x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
@@ -605,7 +605,7 @@ public class IndexFollowingIT extends CcrIntegTestCase {
         }
 
         PutFollowAction.Request followRequest = putFollow("index1", "index2");
-        followRequest.getParameters().setMaxReadRequestSize(new ByteSizeValue(1, ByteSizeUnit.BYTES));
+        followRequest.getParameters().setMaxReadRequestSize(new ByteSizeValue(randomIntBetween(1, 1024), ByteSizeUnit.BYTES));
         followerClient().execute(PutFollowAction.INSTANCE, followRequest).get();
 
         final Map<ShardId, Long> firstBatchNumDocsPerShard = new HashMap<>();
@@ -618,7 +618,7 @@ public class IndexFollowingIT extends CcrIntegTestCase {
             }
         }
 
-        assertBusy(assertTask(1, firstBatchNumDocsPerShard));
+        assertBusy(assertTask(1, firstBatchNumDocsPerShard), 60, TimeUnit.SECONDS);
         for (int i = 0; i < numDocs; i++) {
             assertBusy(assertExpectedDocumentRunnable(i));
         }

